zstacklib/zstacklib/utils/pci.py

```
@@ -419,41 +455,41 @@ def _query_vm_pci_address_mapping(vm_uuid, aliases, alias_to_host, mapping_build
-                # Match alias and build mapping
-                if current_device_info:
-                    for alias in aliases:
-                        if '"{}"'.format(alias) in line:
-                            key, value = mapping_builder(current_device_info, alias, alias_to_host)
-                            mapping[key] = value
-                            # Reset current_device_info after successful match
-                            current_device_info = None
-                            break
-    except Exception as ex:
-        logger.debug("Error querying info pci for VM {}: {}".format(vm_uuid, str(ex)))
+        except Exception as ex:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)